test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Steps/HttpClientSteps.cs

```diff
@@ -14,6 +15,7 @@ namespace NHSD.BuyingCatalogue.Documents.API.IntegrationTests.Steps
     [Binding]
     internal class HttpClientSteps
     {
+        private const string SampleDataPath = "SampleData";
```

Comment:
Currently, `SampleDataPath` is only used in `GetDocumentAsStreamForSolution` so it might be better as a local constant or do you think it's likely that it will eventually be used in other methods too?

Reply:
It might eventually become useful for other methods, however  it might not and for now it isn't, so I think it makes sense to have it as a local const. and when time comes, refactor it.